odoo/tools/config.py

```diff
@@ -533,7 +533,7 @@ def die(cond, msg):
         self.options['translate_modules'].sort()
 
         dev_split = opt.dev_mode and  [s.strip() for s in opt.dev_mode.split(',')] or []
-        self.options['dev_mode'] = 'all' in dev_split and dev_split + ['pdb', 'reload', 'qweb', 'werkzeug', 'xml'] or dev_split
+        self.options['dev_mode'] = 'all' in dev_split and dev_split + ['reload', 'qweb', 'werkzeug', 'xml'] or dev_split
```

Comment:
is werkzeug still in this list on purpose? (and the pre 2.4 ternary too?)
```suggestion
        self.options['dev_mode'] = 'all' in dev_split and dev_split + ['reload', 'qweb', 'xml'] or dev_split
```